splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1645,4 +1645,51 @@ public void addCatalogVersion(TransactionController tc) throws StandardException
 
         }
     }
+
+    public void addMinRetentionPeriodColumn(TransactionController tc) throws StandardException {
```

Comment:
Adding a column should follow this example:
https://github.com/splicemachine/spliceengine/pull/3820/commits/3cbd29dc5bd1fbf79b24f6e9f453e94165a5cb4f
See DB-9745

Reply:
This is an [old PR](https://github.com/splicemachine/spliceengine/pulls?q=10045+) that was already approved by Jun.